tezos/wrapper/src/service.rs

```diff
@@ -783,7 +791,7 @@ impl ProtocolController {
         &self,
         request: ProtocolRpcRequest,
     ) -> Result<ProtocolRpcResponse, ProtocolServiceError> {
-        self.call_protocol_rpc_internal(ProtocolMessage::ProtocolRpcCall(request))
+        self.call_protocol_rpc_internal(request.clone(), ProtocolMessage::ProtocolRpcCall(request))
```

Comment:
@tizoc 
I am afraid of cloning the whole request,
why not just clone path/method instead of the whole request data?
I dont know, if to avoid any cloing is possible here

Reply:
@bkontur we only need the path, no need for the rest, it was just a quick hack to get some meaningful output, will fix in a bit.